build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
heh, sorry, fingers weren't keeping up with my brain.

call it `build-images.sh` like we have `build-go.sh`.